airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/client/AirbyteClient.kt

```diff
@@ -0,0 +1,6 @@
+package io.airbyte.cdk.load.client
+
+abstract class AirbyteClient {
```

Comment:
Can we call this destination client or something? It also feels kind of premature to make this an interface / abstract class, but I get this is for demonstration purposes.